src/core_plugins/kibana/server/lib/manage_uuid.js

```diff
@@ -0,0 +1,73 @@
+import uuid from 'node-uuid';
+import Promise from 'bluebird';
+import { hostname } from 'os';
+import { join as pathJoin } from 'path';
+import { readFile as readFileCallback, writeFile as writeFileCallback } from 'fs';
+
+const serverHostname = hostname();
```

Comment:
I feel like this should be using the `server.info.host`, which is set with `server.config().get('server.host')`